src/node.rs

```diff
@@ -184,11 +112,26 @@ impl fmt::Debug for ItemIds<'_> {
     }
 }
 
-#[derive(Debug, Clone)]
-pub struct SplitPlaneNormal<'a> {
+pub struct SplitPlaneNormal<'a, D: Distance> {
     pub left: NodeId,
     pub right: NodeId,
-    pub normal: Cow<'a, UnalignedF32Slice>,
+    pub normal: Cow<'a, UnalignedVector<D::VectorCodec>>,
+}
+
+impl<D: Distance> fmt::Debug for SplitPlaneNormal<'_, D> {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("SplitPlaneNormal")
+            .field("left", &self.left)
```

Comment:
Can you add the `D::name` under a `distance_name` field?

Reply:
Done in https://github.com/meilisearch/arroy/pull/82/commits/af598ca76e900dba23b0966e8078eeb40b0e9da2